projects/diu-component-library/src/lib/_components/dashboard-population/dashboard-population.component.ts

```diff
@@ -369,13 +369,8 @@ export class DashboardPopulationComponent implements OnInit {
     createCharts() {
         setTimeout(() => {
             this.createDeprivationScale(this.DDimension, this.DDimGroup);
-            let endF = this.AgeDimGroupF.top() * 5; // each age is banded into 5
-            let endM = this.AgeDimGroupM.top() * 5; // each age is banded into 5
-            const totalCitizens = this.all.value();
-            if (totalCitizens && totalCitizens < 1000000) {
-                endF = totalCitizens / 20;
-                endM = totalCitizens / 20;
-            }
+            const endF = this.AgeDimGroupF.top() * 5; // each age is banded into 5
+            const endM = this.AgeDimGroupM.top() * 5; // each age is banded into 5
```

Comment:
can you figure out the reason the totalCitizens check was added? reverting this code back to its original will cause another issue, please review and come up with a suitable check so that the graphs draw correctly on each filter